docs/2_new_data_model.md

```diff
@@ -22,7 +22,7 @@ In this note, we give an example for converting the data into KITTI format.
 
 **Note**: We take Waymo as the example here considering its format is totally different from other existing formats. For other datasets using similar methods to organize data, like Lyft compared to nuScenes, it would be easier to directly implement the new data converter (for the second approach above) instead of converting it to another format (for the first approach above).
 
-### KITTI dataset format
+## KITTI dataset format
```

Comment:
Just keep it as a subtitle under "Prepare the customized dataset". It is only an example under that instead of a parallel section.